File-Formats/Presentation/FAQ.md

```diff
@@ -48,6 +48,12 @@ Current version of the Essential Presentation library does not provide support f
 
 You can improve the quality of converted images by specifying the image resolution. Refer – [Converting PowerPoint presentation to Images](/file-formats/presentation/getting-started#converting-powerpoint-presentation-to-images)
 
+## Why Metafile images doesn't gets preserved in Presentation to PDF/Image conversion?
+
+In .NET Core targeting applications, metafile images such as EMF and WMF have some limitations. Presentation library used SkiaSharp graphics library in Syncfusion.PresentationRenderer.Net.Core to render the text and images in PDF/Image conversion. But SkiaSharp doesn’t support the metafile images such as EMF and WMF format images, so, it doesn’t preserve the EMF images in ASP.NET Core platform. So, those images will not preserve in Presentation document to Image/PDF conversion using Presentation library.
```

Comment:
In .NET Core targeting applications, metafile images such as EMF and WMF have some limitations. The presentation library used SkiaSharp graphics library in Syncfusion.PresentationRenderer.Net.Core to render the text and images in PDF/Image conversion. But SkiaSharp does not support metafile images such as EMF and WMF format images, so it does not preserve the EMF images in the ASP.NET Core platform. So, those images will not be preserved in the Presentation document to Image/PDF conversion using the Presentation library.

Reply:
Modified